### ISSUE TITLE
Not available on first play

### ISSUE THREAD

**Issue by bdunlay** [fabiofalci/sconsify#55]
I'm noticing that often when I choose a new song, or a new playlist, the songs are "Not available" -- if I choose it again, it suddenly becomes available. Seems like it is erroneously reporting that message while it's fetching the song.

**Comment by fabiofalci** [comment]
Yeah, I've seen that before. It seems it takes some time to load so it can't reproduce immediately but need to wait a little bit.

I've add a delay when it first get the error but it seems is not working properly then.
